fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeActions.java

```diff
@@ -33,7 +34,7 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-public class PrivilegeActions implements SystemAccessControl {
+public class PrivilegeActions implements AccessControl {
```

Comment:
Shall we unify the naming, `AccessControl` and `AstVisitor ` are the base classes, so we will have `NativeAccessControl`(to replace `PrivilegeActions`) and `NativePrivilegeChecker`, `NativePrivilegeChekcerVisitor`. Later we can have `RangerAccessControl`, `RangerPrivilegeChecker`...

Reply:
Currently, `PrivilegeXXX` without prefix is too general and may confuse the developers.